testSetup.js

```diff
@@ -1 +1,5 @@
+/* eslint-env node */
 import 'jest-axe/extend-expect';
+import filterXss from './app/assets/javascripts/lib/xss';
+
+global.filterXss = filterXss;
```

Comment:
Wait should this be `global.filterXSS` capital `S`?

Reply:
I think so.

cc @nickytonline: I don't think it's case insensitive, the code calls `filterXSS`, not `filterXss`

Reply:
If you look where we use it in the code it's lower case.

Reply:
@nickytonline I'm confused then:

```shell
➜  forem git:(master) rg filterXss
testSetup.js
3:import filterXss from './app/assets/javascripts/lib/xss';
5:global.filterXss = filterXss;
➜  forem git:(master) rg filterXSS | wc -l
      26
```

`filterXss` is only inside `testSetup.js` but `filterXSS` is used 26 times across the website, tests included. Are they two different things?

Reply:
@rhymes, testSetup.js makes it available globally to all tests. Previously it was being mocked in tests and then I realized it didn't need to be.

The import here is literally just for tests.